app/auth/UnprotectedPageController.scala

```diff
@@ -10,10 +12,21 @@ import javax.inject.Inject
 class UnprotectedPageController @Inject ()(val controllerComponents: SecurityComponents) extends Security[CommonProfile] {
 
   implicit class RequestUtils(request: Request[AnyContent]) {
+    val webContext = new PlayWebContext(request, playSessionStore)
+    val profileManager = new ProfileManager[CommonProfile](webContext)
```

Comment:
Because the webContext is coming from the request, I don't think you can have it at the class level like this. I've tried it against the /faq page while logged in and after a few refreshes, the profile starts to come back as Option.empty. I'm not 100% sure why but if you create the webContext each time inside the two methods then it works as expected. You can put those lines into a method though, something like 
```
private def getProfile = {
      val webContext = new PlayWebContext(request, playSessionStore)
      new ProfileManager[CommonProfile](webContext)
    }
```